backend/src/main/java/com/benine/backend/camera/CameraController.java

```diff
@@ -24,14 +26,10 @@
 
   private int highestIdInUse = 1;
 
-  private ServerController serverController;
-
   /**
    * Create a new CameraController.
-   * @param serverController to interact with the server.
    */
-  public CameraController(ServerController serverController) {
-    this.serverController = serverController;
+  public CameraController() {
```

Comment:
It's a very tiny detail but we can remove this constructor at all since it now is the default constructor.